docs/quickstart-cli.md

```diff
@@ -0,0 +1,168 @@
+# Verify a artifact graph on commandline 
+
+## Setup
+
+- Build the `ratify` binary
+
+```bash
+git clone https://github.com/deislabs/ratify.git
+go build -o ~/bin ./cmd/ratify
```

Comment:
this assumes there is a dir called `bin`. if it doesn't, it'll create the binary called `dir`

Reply:
also, `~/bin` might not be in `PATH`